elevate-cpanel

```diff
@@ -5152,6 +5153,21 @@ EOS
         return;
     }
 
+    sub _fixup_epel_repo ($self) {
+
+        File::Copy::mv(
+            Elevate::Constants::YUM_REPOS_D . '/epel.repo',
+            Elevate::Constants::YUM_REPOS_D . '/epel.o.repo'
+        ) or die "Failed to rename epel.repo: $!";
+
+        $self->yum->reinstall('epel-release');
```

Comment:
is epel-release always a part of a repo?

Reply:
First verifying that the package is installed should allow us to attempt a re-install.